### PR TITLE
fix: vue in edit grammars

### PR DESCRIPTION
**Pull request by ilevyor**
edit grammars didn't handle vue

### CODE REVIEW

**Comment by morgante** [comment]
> edit grammars didn't handle due

vue...?

File: resources/edit_grammars.mjs [hunk 1]
```diff
@@ -234,6 +235,31 @@ async function buildLanguage(language) {
 
     await copyWasmParser('typescript', 'tree-sitter-typescript/');
     await copyWasmParser('tsx', 'tree-sitter-typescript/');
+  } else if (language === 'vue') {
+    // typescript is special
```
Comment (morgante) [inline resources/edit_grammars.mjs:239]: Is this about TypeScript or is it about Vue?

**Review by morgante — CHANGES_REQUESTED** [review]
Please self-review your own work. This is littered with typos and copy/paste.

**Review by morgante — APPROVED** [review]

**Comment by nicooprat** [comment]
Any ETA when this will be released? Thanks 🙏

**Comment by morgante** [comment]
Hi @nicooprat, this was in the last release so JavaScript patterns should now target Vue.

**Pull request merged**